src/main/java/appeng/items/parts/ItemPart.java

```diff
@@ -297,22 +298,40 @@ public String getUnlocalizedGroupName( final Set<ItemStack> others, final ItemSt
 							group = true;
 						}
 						break;
+					case FLUID_IMPORT_BUS:
+						importBusFluids = true;
+						if( u == pt )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS134)